params/default.go

```diff
@@ -28,3 +33,10 @@ func DefaultBootstrappersInfos() []peer.AddrInfo {
 func DefaultBootstrappers() []string {
 	return bootstrapList[defaultNetwork] // network is guaranteed to be valid
 }
+
+func init() {
+	if genesis, ok := os.LookupEnv("CELESTIA_PRIVATE_GENESIS"); ok {
```

Comment:
Can we also document this as an example somewhere? Or do you rather expect anyone who wants to run a private network to read the godoc comment of `GenesisFor`?

Reply:
@liamsi, WDYM? Like a `something_example.go`?
> as an example somewhere

I expect this but am happy to document this somewhere as well. Ideally, our docs website under the special section about running your own celestia-node networks. cc @YazzyYaz
> Or do you rather expect anyone who wants to run a private network to read the godoc comment of GenesisFor?

Reply:
Maybe simply add an example command-line usage somewhere.

Reply:
https://github.com/celestiaorg/celestia-node/issues/574

Reply:
Have no idea where to put such clip example right now. Making a special document for it right now is overkill.